### PR TITLE
Add playbook instructions for creating a team via the API

### PR DESCRIPTION
**Pull request by tony-griffin**
Add instructions for creating a new team in either the CFS or ASC services, using Postman, via the social-care-case-viewer-api

### CODE REVIEW

**Review by acodeninja — APPROVED** [review]
A little over-complicated, but seems reasonable.

File: docs/playbook/create_team.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+- select `“stages”`
+
+- select `"mosaic-prod”`
+
+- where is says `“Invoke URL:"`
+  copy the value of the URL
+
+```
+https://thecorrecturl.something.yourregion.amazonaws.com/production
```
Comment (LBHTKarki) [inline docs/playbook/create_team.md:171]: stage is called `mosaic-prod` instead of `production`

Comment (tony-griffin) [reply docs/playbook/create_team.md:171]: Yes I put this as "production" as the whole RL is made up, I didn't want to leak any of the details that were in AWS.
Do you think it's misleading?

Comment (LBHTKarki) [reply docs/playbook/create_team.md:171]: I think it's fine to leave as is i.e. production. I automatically replaced the dummy url with the real one leaving just the stage

Comment (tony-griffin) [reply docs/playbook/create_team.md:171]: Cool no worries, thanks for the feedback.

Comment (acodeninja) [reply docs/playbook/create_team.md:171]: You could also wrap parts that require replacement by users in `<>` so `https://<full url from api gateway>/` or `https://<api gateway id>.execute.<aws region>.amazonaws.com/<environment>`.

File: docs/playbook/create_team.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
+## Add new team to the Social Care Case Viewer API
```
Comment (acodeninja) [inline docs/playbook/create_team.md:1]: You should add some head matter here, so it shows up in the menu, etc.

EG:
```
---
id: index
title: Introduction
slug: /
---

Document contents
```

File: docs/runbooks/case-viewer/every-day/adding-a-new-team.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+The API expects to receive a JSON object containing the data properties needed to create a new team.
+
+These properties consist of a `name` and a `context`
+
+name = the name of the new team
+
+context = refers to which service, 
+
+“A” for ASC or “C” for CFS
```
Comment (acodeninja) [inline docs/runbooks/case-viewer/every-day/adding-a-new-team.md:54]: Sorry to push back here, but can you either use the `abbr` tag here to explain the acronyms, or better yet get rid of them all together.

For reference https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr

Comment (tony-griffin) [reply docs/runbooks/case-viewer/every-day/adding-a-new-team.md:54]: Removed

**Pull request merged**